gui/src/components/SuiteForm.js

```diff
@@ -0,0 +1,34 @@
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+
+function SuiteForm() {
+  const tests = [{
+      name: "cool-test",
+      label: "Cool Test"
+    }, {
+      name: "epic-test",
+      label: "Epic Test"
+    }
+  ];
+
+  return (
+    <Form className="w-25 py-2 px-4 bg-secondary text-white d-flex flex-column justify-content-between">
+      <Form.Group>
+        <Form.Label>URL</Form.Label>
+        <Form.Control type="text" placeholder="google.com"/>
+      </Form.Group>
+
+      <Form.Group>
+        {tests.map((test, index) => (
+          <Form.Group key={index} className="bg-primary bg-opacity-50 rounded border border-dark py-2 px-4 my-2">
+            <Form.Check className="d-flex justify-content-between" type="checkbox" name={test.name} label={test.label}/>
+          </Form.Group>
+        ))}
+      </Form.Group>
+      
+      <Button variant="outline-light" className="w-100">Submit</Button>
```

Comment:
Maybe I'm just dumb but what does this do?

Reply:
So the tests will always fail because this MR does not implement the suite form so the input enter is not used.

The 'connect & send' button issue related to the backend system, there's a bug that needs to be investigated.

Reply:
Ahh makes sense, maybe hold off this MR until the bug is sorted

Reply:
I don't think that's a very good idea, the bug is completely unrelated to this MR.

Reply:
fair enough

Reply:
Well in this MR the submit button does nothing, the 'suite form' MR adds it's proper functionality.